host-configs/rzgenie-toss_3_x86_64_ib-clang@10.0.0.cmake

```diff
@@ -69,7 +69,7 @@ set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "")
 
 # Root directory for generated TPLs
 
-set(TPL_ROOT "/usr/WS1/axom/libs/toss_3_x86_64_ib/2021_05_05_17_27_26/clang-10.0.0" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/toss_3_x86_64_ib/2021_05_10_21_44_53/clang-10.0.0" CACHE PATH "")
```

Comment:
@white238 -- Do you think it would make sense to have `TPL_ROOT` be one directory higher and then use it for the spack compilers as well?

That would reduce the diffs when we generate a new host-config.

Reply:
The compiler lines are output by the spack build package class.  It would be kind of a hassle and very us specific to do that.